src/components/bottomTabBar/view/bottomTabBarStyles.js

```diff
@@ -6,7 +6,7 @@ const deviceWidth = Dimensions.get('window').width;
 
 export default EStyleSheet.create({
   wrapper: {
-    width: '$deviceWidth',
+    width: deviceWidth,
```

Comment:
can you remove `const deviceWidth = Dimensions.get('window').width;` instead of this changes

Reply:
`deviceWidth` is being used in Line 25, that's why I kept it and changed about to local deviceWidth as well so that we don't have to travel back to get it from global. If you can suggest fix for Line 25, then we can completely remove Line 5

Reply:
hmm okay i didnt see that. i just reviewed 25. line, we cannot change to that line so you can use it is okay